docs/ci-cd-environment/aws-support.md

```diff
@@ -0,0 +1,112 @@
+---
+Description: This guide describes how to set up a fleet of self-hosted agents in AWS
+---
+
+# AWS support
+
+!!! beta "Self-hosted agents - closed beta"
+    Self-hosted agents are in closed beta. If you would like to run Semaphore agents in your infrastructure, please [contact us and share your use case](https://semaphoreci.com/contact). Our team will get back to you as soon as possible.
+
+Even though manually starting and stopping agents might serve your purposes, having a dynamically sized pool of agents which automatically resizes itself on high and low load periods allows you to have a more highly available fleet of agents at your disposal. 
```

Comment:
The intro starts off in a defensive tone and meanders a bit before landing to the point finally in the third paragraph. The purpose of the intro is to help the reader understand whether they are on the right page. So I recommend cutting the first two paragraphs completely and getting to the point immediately.